integration-tests/postgresql_test.go

```diff
@@ -217,6 +223,10 @@ var _ = Describe("Postgresql", Label("Postgresql"), func() {
 			Entry(nil, "backup_window", "01:02-03:04"),
 			Entry(nil, "copy_tags_to_snapshot", false),
 			Entry(nil, "delete_automated_backups", false),
+			Entry(nil, "monitoring_interval", 30),
```

Comment:
not sure why monitoring interval is changing in this test. Just pointing it out in case it is an undesired change.

Reply:
I realised that it wasn't being tested the modification of that property and I added it in the same PR. I am sorry for mixing concepts